src/core/components/operation-summary-method.jsx

```diff
@@ -19,7 +19,9 @@ export default class OperationSummaryMethod extends PureComponent {
     } = this.props
 
     return (
-      <span className="opblock-summary-method">{method.toUpperCase()}</span>
```

Comment:
Structure change due to limitation in implementing `em` units for spacing as well as fonts.
if an element has `font-size: 14em` and `margin: 2em` the margin `em` will reference the font-size of it's own element - not the next parent! To counter this we need to create a inner element to apply the font-size and the outer element to apply spacing